Sources/idt/idt.cc

```diff
@@ -291,13 +306,39 @@ class visitor : public clang::RecursiveASTVisitor<visitor> {
     return false;
   }
 
+  // Emit a FixIt if a symbol is annotated with a default visibility or DLL
+  // export/import annotation. The FixIt will remove the annotation
+  template <typename Decl_>
+  void check_symbol_not_exported(const Decl_ *D, const std::string &message) {
+    clang::SourceLocation SLoc;
+    if (const auto *VA = D->template getAttr<clang::DLLExportAttr>())
```

Comment:
I think that `A` is more appropriate as the initialism - as it is an `Attribute`.